poetry_multiproject_plugin/components/project/prepare.py

```diff
@@ -34,4 +34,4 @@ def copy_project(project_file: Path, destination: Path) -> Path:
 
 
 def normalize_top_namespace(namespace: Union[str, None]) -> Union[str, None]:
-    return re.sub("[^a-zA-Z_]", "", namespace) if namespace else None
+    return re.sub("[^a-zA-Z_/]", "", namespace.strip("/")) if namespace else None
```

Comment:
👏

Reply:
i went back and forth on removing the `/` from before/after or just throwing an error and having the user change it. not certain which one is better but i opted for this as it's "easier". :)